packages/contentful/package.json

```diff
@@ -14,6 +14,7 @@
     "space:migrate:crn": "yarn ctf-migrate up --all",
     "space:migrate:crn:dryrun": "yarn ctf-migrate up --all --dry-run",
     "space:migrate:crn:count": "yarn ctf-migrate list --all | grep \\\\[pending\\\\] | wc -l",
+    "generate:migration:crn": "contentful space generate migration -s $CONTENTFUL_SPACE_ID -e $CONTENTFUL_ENV_ID",
```

Comment:
I thought we were using this command `ctf-migrate create ...` as per https://github.com/yldio/asap-hub/blob/602236c1624ecbc2d8b640526251d98bc9c1ba2b/README.md#L206